src/java.xml/share/classes/com/sun/org/apache/xalan/internal/lib/ExsltDatetime.java

```diff
@@ -97,7 +101,20 @@ public static String dateTime()
         char posneg = hrs < 0? '-': '+';
         buff.append(posneg).append(formatDigits(hrs)).append(':').append(formatDigits(min));
       }
-      return buff.toString();
+      return buff.toString();*/
```

Comment:
This commented out block is  difficult to see given Can we clean this up or better yet remove this code if not needed?

Reply:
Thanks. Removed.